pkg/api/v1alpha1/cluster_types.go

```diff
@@ -275,6 +276,14 @@ type ControlPlaneConfiguration struct {
 	SkipLoadBalancerDeployment bool `json:"skipLoadBalancerDeployment,omitempty"`
 }
 
+// MachineHealthCheck allows to configure timeouts for machine health checks.
+type MachineHealthCheck struct {
+	// NodeStartupTimeout overrides the the default node startup timeout in machine health checks.
+	NodeStartupTimeout string `json:"nodeStartupTimeout,omitempty"`
```

Comment:
Should we use [`metav1.Duration`](https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#Duration)?
I also, I think these two should probably be pointers, right? Because they are both optional and independent from each other.